src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineInstance.java

```diff
@@ -80,7 +82,11 @@ protected void _terminate(TaskListener listener) throws IOException, Interrupted
 
 
     }
-    
+
+    public boolean getCreateSnapshot() {
```

Comment:
This public method is missing header javadocs. Also, it's typical java convention to use the "is" prefix for methods with boolean return types.